store/tikv/tests/2pc_test.go

```diff
@@ -1031,7 +1030,6 @@ func (s *testCommitterSuite) TestResolvePessimisticLock(c *C) {
 	untouchedIndexKey := []byte("t00000001_i000000001")
 	untouchedIndexValue := []byte{0, 0, 0, 0, 0, 0, 0, 1, 49}
 	noValueIndexKey := []byte("t00000001_i000000002")
-	c.Assert(tablecodec.IsUntouchedIndexKValue(untouchedIndexKey, untouchedIndexValue), IsTrue)
```

Comment:
I remember we can use `tablecodec` in tikv tests?

Reply:
It is ok to remove this line since I've added this test in `tablecodec/tablecodec_test.go`